readme.md

```diff
@@ -15,50 +15,56 @@ cp .env.example .env
 Build app containers:
 
 ```
-make build
+task build
```

Comment:
You have to update readme: how to install task and describe how it works.

You cna base on this:
https://github.com/blumilksoftware/environment/blob/main/readme.md